package/yast2-sap-ha.spec

```diff
@@ -96,6 +98,10 @@ install -m 755 data/check_ssh.expect %{buildroot}%{yast_dir}/data/sap_ha/
 # Augeas lens for SAP INI files
 install -m 644 data/sapini.aug %{buildroot}%{augeas_dir}
 
+%post
+/usr/bin/systemctl enable csync2.socket
+/usr/bin/systemctl start  csync2.socket
+
```

Comment:
I'm not sure if it makes sense to enable and start the service *always*. The usual approach is to enable the needed service when saving the configuration by the YaST module.

And IMHO this is also bad from the security POV, it runs the service whenever you install this YaST package, even if you possibly do not need that. IIRC only very few services are allowed to run by default, the security team might complain about this...

Reply:
Ah I see...

What about some small additional YaST module which would be run on the secondary node? It would just enable and start that service. Or just document this requirement?

Anyway, if there is no better solution then I think it needs to be consulted with the security team as they want to keep the list of default running services as small as possible. This might be a security problem...